be/src/olap/rowset/segment_v2/ordinal_page_index.h

```diff
@@ -97,6 +98,19 @@ class OrdinalPageIndex {
     }
     OrdinalPageIndexIterator end() {
         return OrdinalPageIndexIterator(this, _num_pages);
+	}
+	rowid_t get_first_row_id(int page_index) const {
+        return _rowids[page_index];
+    }
+
+    rowid_t get_last_row_id(int page_index, size_t column_row_count) const {
```

Comment:
you can write last row id to _rowids, if you do that, you can make this logic simple.

just get_first_row_id(next_page_index) - 1, and without column_row_count param

Reply:
ok